app/services/mapping/governmentagencygoodsitem/StatisticalValueAmountBuilder.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services.mapping.governmentagencygoodsitem
+
+import models.declaration.governmentagencygoodsitem.{Amount, GovernmentAgencyGoodsItem}
+import services.cache.ExportItem
+import wco.datamodel.wco.declaration_ds.dms._2.GovernmentAgencyGoodsItemStatisticalValueAmountType
+import wco.datamodel.wco.dec_dms._2.Declaration.GoodsShipment.{
+  GovernmentAgencyGoodsItem => WCOGovernmentAgencyGoodsItem
+}
+
+object StatisticalValueAmountBuilder {
```

Comment:
Similar to above, can we make this a class and inject the dependent builders

Reply:
this is TODO as well as the PR title suggested this is PART1